ansible_ai_connect/ai/api/views.py

```diff
@@ -176,9 +176,18 @@
 
 
 class AACSAPIView(APIView):
+    def __init__(self):
+        super().__init__()
+        self.event = None
+        self.model_id = None
+        self.exception = None
+
     def initialize_request(self, request, *args, **kwargs):
+        # Call super first to ensure request object is correctly
+        # initialised before instantiating event
+        _request = super().initialize_request(request, *args, **kwargs)
```

Comment:
I've a light objection regarding the use of the `_`, if we want to flag our private variables like this and be consistent, we will need to re-factorize the whole code base. I would prefer we keep the current approach (no `_` prefix).

Reply:
Renamed.